metron-platform/metron-common/src/main/java/org/apache/metron/common/dsl/functions/resolver/ClasspathFunctionResolver.java

```diff
@@ -65,24 +72,48 @@
  *
  */
 public class ClasspathFunctionResolver extends BaseFunctionResolver {
```

Comment:
The old one is much slower than this one and this one completely replaces it for the purposes that it was intended for (resolving stellar functions from within the Metron project).  This isn't so much an extension as an expansion and a replacement.